root/static/scripts/edit/URLCleanup.js

```diff
@@ -4690,6 +4690,34 @@ const CLEANUPS: CleanupEntries = {
       return {result: false, target: ERROR_TARGETS.URL};
     },
   },
+  'threads': {
+    match: [new RegExp('^(https?://)?([^/]+\\.)?threads\\.net/', 'i')],
+    restrict: [{...LINK_TYPES.streamingfree, ...LINK_TYPES.socialnetwork}],
+    clean: function (url) {
+      return url.replace(
+        /^(?:https?:\/\/)?(?:www\.)?threads\.net(?:\/#!)?\//,
+        'https://www.threads.net/',
+      );
+    },
+    validate: function (url, id) {
+      const isAProfile = /^https:\/\/www\.threads\.net\/(@[^/]+)$/.test(url);
```

Comment:
Capturing group is not needed here:
```suggestion
      const isAProfile = /^https:\/\/www\.threads\.net\/@[^/]+$/.test(url);
```